File: TD/TD3/Code/P1.cpp
/**
 * Dessiner le motif donné
 */
#include <iostream>

void dessin(int lngr, int rep)
{
  for (int i = 0; i < rep; ++i)
    for (int j = 1; j <= lngr; ++j) {
      for (int k = 0; k < j; ++k)
        std::cout << "*" << std::flush;
      std::cout << std::endl;
    }
}

int main()
{
  int lngr;
  int rep;

  std::cout << "Donne moi la longueur du motif : " << std::flush;
  std::cin >> lngr;
  std::cout << "Donne moi le nombre de répétition : " << std::flush;
  std::cin >> rep;

  dessin(lngr, rep);

  return 0;
}

File: TD/TD3/Code/Python/P1.py
#!/usr/bin/python
# Dessiner le motif donné


def dessine(lngr, rep):
    for i in range(rep):
        for j in range(1, lngr + 1):
            for k in range(j):
                print('*', end='', sep='')
            print('\n', end='')


print('Donne moi la longueur de motif : ', end='')
lngr = int(input())
print('Donne moi le nombre de répétition : ', end='')
rep = int(input())

dessine(lngr, rep)
